QuizApp/App/Screens/Quiz/QuizCell.swift

```diff
@@ -0,0 +1,106 @@
+import UIKit
+
+final class QuizCell: UICollectionViewCell {
+
+    var data: Category? {
+        didSet {
+            guard let data = data else { return }
+            update(data: data)
+        }
+    }
+
+    private var imageView: UIImageView!
+    private var titleLabel: UILabel!
+    private var descriptionLabel: UILabel!
+    private var difficultyView: DifficultyView!
+    static let reuseIdentifier = "QuizCell"
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+}
+
+// MARK: - ConstructViewsProtocol methods
+extension QuizCell: ConstructViewsProtocol {
+
+    func createViews() {
+        imageView = UIImageView()
+        contentView.addSubview(imageView)
+
+        titleLabel = UILabel()
+        contentView.addSubview(titleLabel)
+
+        descriptionLabel = UILabel()
+        contentView.addSubview(descriptionLabel)
+
+        difficultyView = DifficultyView()
+        contentView.addSubview(difficultyView)
+    }
+
+    func styleViews() {
+        contentView.backgroundColor = .white.withAlphaComponent(0.3)
+        contentView.layer.cornerRadius = 20
+
+        imageView.image = UIImage(named: "imgSport")
+        imageView.contentMode = .scaleAspectFill
+        imageView.backgroundColor = .yellow
+        imageView.layer.cornerRadius = 10
+        imageView.layer.masksToBounds = true
+
+        titleLabel.textColor = .white
+        titleLabel.font = Fonts.sourceSansProBold24.font
+
+        descriptionLabel.text = "Quiz description that can\nusually span over multiple \nlines"
+        descriptionLabel.textColor = .white
+        descriptionLabel.numberOfLines = 0
+        descriptionLabel.font = Fonts.sourceSansProSemiBold14.font
+    }
+
+    func defineLayoutForViews() {
+        imageView.snp.makeConstraints {
+            $0.top.equalTo(contentView.safeAreaLayoutGuide).offset(20)
```

Comment:
Please chain constraints that have the same relation & offset:
`$0.top.leading.bottom.equalTo(contentView.safeAreaLayoutGuide).inset(20)`
`$0.width.height.equalTo(100)`